uses_cases/auth_cases.py

```diff
@@ -0,0 +1,45 @@
+from operations.user_operations import *
+from models.user import *
+from pprint import pprint
+from dependencies import user_deps, responses, qr_deps, token_deps
+from fastapi import HTTPException
+import time
+from bson.objectid import ObjectId
+from datetime import datetime, timedelta
+
+
+def validation_login_auth(data: auth_in):
+    user_retrive = retrive_user({"email": data.email})
+    if user_retrive:
+        pprint(user_retrive)
+        is_equal = user_deps.verify_passowrd(
+            data.password, user_retrive['hashed_password'])
+        if is_equal:
+            return responses.response_model({'key_qr': user_retrive['key_qr'], 'email': user_retrive['email']}, "successfull")
+
+        raise HTTPException(status_code=404, detail=responses.error_response_model(
+            'password not is equal', 404, 'Error'))
+
+    raise HTTPException(status_code=404, detail=responses.error_response_model(
+        'user not exist.!', 404, 'Error'))
```

Comment:
Closing parenthesis in a new line.

"user not exists.!" should be "user doesn't exist!"  (be aware of revoming the point before the exclamation mark.